### PR TITLE
FIX: Match Values between $line-heights and $font-sizes

### PR DESCRIPTION
**Pull request by talbs**
This work syncs up canned value names between ``$font-sizes`` and ``$line-heights`` Sass Maps

### CODE REVIEW

**Comment by talbs** [comment]
@andywaldrop, mind reviewing this fix to the good issue you noted? I'll point you to where I'm syncing the values.

File: _src/pattern-library/sass/utilities/_variables.scss [hunk 1]
```diff
@@ -199,8 +199,8 @@ $font-sizes: (
     large:          21,
     mid-large:      18,
     base:           16,
-    small:          14,
-    x-small:        12,
+    mid-small:      14,
```
Comment (talbs) [inline _src/pattern-library/sass/utilities/_variables.scss]: @andywaldrop, boom! Here's where I'm making things more alike.

**Comment by andywaldrop** [comment]
@talbs larger discussion, but I was more tempted to use small, x-small, xx-small for mobile font sizes as I need to go lower in value.

**Comment by talbs** [comment]
@andywaldrop, understood. I think we can still accommodate those smaller sizes, but I'd rather stick with  the consistent mid-* step (since we use it on the larger side of the scale). You cool with if we stay with this direction?

```scss
// typography: sizes
$font-sizes: (
    xxx-large:      38,
    xx-large:       28,
    x-large:        24,
    large:          21,
    mid-large:      18,
    base:           16,
    mid-small:      14,
    small:          12,
    x-small:          10,
    xx-small:          8,  // demo for my amigo, Andy - real value can change
);

// typography: line-height
$line-heights: (
    xx-large:       1.40,
    x-large:        1.40,
    large:          1.50,
    mid-large:      1.60,
    base:           1.5,
    mid-small:      1.60,
    small:          1.5,
    x-small:          1.25,   // demo for my amigo, Andy - real value can change
    xx-small:          1,   // demo for my amigo, Andy - real value can change
    none:           0
);
```

**Comment by talbs** [comment]
@andywaldrop, per our in person chat. I've synced up both line-heights and font-sizes to not use any ``mid-*`` values. Its straight base + small/large + x-sizing. Here's the updated scale (with your mobile small scale plumbed through):

```scss
// typography: sizes
$font-sizes: (
    xxxx-large:     38,
    xxx-large:      28,
    xx-large:       24,
    x-large:        21,
    large:          18,
    base:           16,
    small:          14,
    x-small:        12,
    xx-small:       11,
    xxx-small:      10,
);

// typography: line-height
$line-heights: (
    xxx-large:      1.40,
    xx-large:       1.40,
    x-large:        1.50,
    large:          1.60,
    base:           1.5,
    small:          1.60,
    x-small:        1.5,
    none:           0
);
```

**Comment by talbs** [comment]
@andywaldrop, let me know if there's anything I can do to sync things up as well as when you're satisfied.

**Comment by andywaldrop** [comment]
:+1: I have no idea if this works

**Pull request merged**